apps/sms/test/marionette/new_activity_test.js

```diff
@@ -102,7 +102,7 @@ marionette('Messages as "new" activity target', function() {
         return !!window.wrappedJSObject.Compose;
       }), 'XSS should not be performed');
 
-      assert.isTrue(messagesApp.Composer.sendButton.enabled());
+      client.waitFor(() => messagesApp.Composer.sendButton.enabled());
```

Comment:
Thanks for the explanation. I think that deciding when the compose is ready might not be difficult, but considering the text input at the same time is much more complex. Current solution is good enough for me.